solvers/BMI/BMIspotless.m

```diff
@@ -0,0 +1,170 @@
+classdef BMIspotless < spotsosprog
```

Comment:
would love to make this part of the NonlinearProgram hierarchy (so that we could, e.g. add existing Constraint objects).  but perhaps another day.